templates/CRM/Contribute/Page/Tab.tpl

```diff
@@ -29,55 +29,91 @@
+              {ts 1=$displayName}Contributions received from %1 since inception.{/ts}
             {/if}
-        </div>
+          </div>
 
-        {if $action eq 16 and $permission EQ 'edit'}
+          {if $action eq 16 and $permission EQ 'edit'}
             <div class="action-link">
-                <a accesskey="N" href="{$newContribURL}" class="button"><span><i class="crm-i fa-plus-circle"></i> {ts}Record Contribution (Check, Cash, EFT ...){/ts}</span></a>
-                {if $newCredit}
-                    <a accesskey="N" href="{$newCreditURL}" class="button"><span><i class="crm-i fa-credit-card"></i> {ts}Submit Credit Card Contribution{/ts}</span></a>
-                {/if}
-                <br /><br />
+              <a accesskey="N" href="{$newContribURL}" class="button"><span><i class="crm-i fa-plus-circle"></i> {ts}Record Contribution (Check, Cash, EFT ...){/ts}</span></a>
+              {if $newCredit}
+                <a accesskey="N" href="{$newCreditURL}" class="button"><span><i class="crm-i fa-credit-card"></i> {ts}Submit Credit Card Contribution{/ts}</span></a>
+              {/if}
+              <br /><br />
             </div>
-          <div class='clear'></div>
-        {/if}
+            <div class='clear'></div>
+          {/if}
 
-        {if $rows}
+          {if $rows}
             {include file="CRM/Contribute/Page/ContributionTotals.tpl" mode="view"}
-          <div class='clear'></div>
+            <div class='clear'></div>
             {include file="CRM/Contribute/Form/Selector.tpl"}
-        {else}
+          {else}
             <div class="messages status no-popup">
-                    <div class="icon inform-icon"></div>
-                    {ts}No contributions have been recorded from this contact.{/ts}
+              <div class="icon inform-icon"></div>
+              {ts}No contributions have been recorded from this contact.{/ts}
             </div>
-        {/if}
-
-        {if $recur}
-          <div class="crm-block crm-contact-contribute-recur">
-            <h3>{ts}Recurring Contributions{/ts}</h3>
-            {include file="CRM/Contribute/Page/ContributionRecur.tpl"}
-          </div>
-        {/if}
+          {/if}
 
-        {if $softCredit}
-          <div class="crm-block crm-contact-contribute-softcredit">
-            <h3>{ts}Soft credits{/ts} {help id="id-soft_credit"}</h3>
-            {include file="CRM/Contribute/Page/ContributionSoft.tpl"}
-          </div>
-        {/if}
+          {if $softCredit}
+            <div class="crm-block crm-contact-contribute-softcredit">
+              <h3>{ts}Soft credits{/ts} {help id="id-soft_credit"}</h3>
+              {include file="CRM/Contribute/Page/ContributionSoft.tpl"}
+            </div>
+          {/if}
+        </div>
+        <div id="recurring-subtab" class="ui-tabs-panel ui-widget-content ui-corner-bottom">
+          {if $recur}
+            <div class="crm-block crm-contact-contribute-recur">
```

Comment:
Can we get additional class selectors on the two div classes here please so they can be uniquely identified for theming/jquery customisation  Eg. 
`<div class="crm-block crm-contact-contribute-recur crm-contact-contribute-recur-active">`
`<div class="crm-block crm-contact-contribute-recur crm-contact-contribute-recur-inactive">`